WebContent/login.jsp

```diff
@@ -0,0 +1,29 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>LogIn</title>
+</head>
+<body>
+	<% 
+		String msg = (String) request.getAttribute("error");
+		if(msg==null){
```

Comment:
Please remove boilerplate code..

Reply:
Vai, I am using this code to show the error message if the username or password is wrong.